spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Fabricator(:user) do
+  email { FFaker::Internet.email }
+  password { 'password123' }
+  password_confirmation { 'password123' }
```

Comment:
Maybe this? 💭 

```suggestion
  password 'password123'
  password_confirmation { |attrs| attrs[:password] }
```

Reply:
I wanted to keep it static since I felt like there wasn't really a reason to change this, unless you think otherwise? 🤔 😮